dev/defs/shapes/text.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We need to add these jsons for text, even though we have no 'code' for them in rive-cpp yet?

Reply:
yeah it's just to keep the defs synced up, instead of having another submodule for them